client/gutenberg/extensions/email-subscribe/edit.jsx

```diff
@@ -0,0 +1,47 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { ServerSideRender, TextControl } from '@wordpress/components';
+import { InspectorControls } from '@wordpress/editor';
+/**
+ * Internal dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+const fields = [
+	{ id: 'title', label: __( 'Title', 'jetpack' ) },
```

Comment:
The i18n wrapper functions don't need the last `textdomain` argument.

Reply:
Done, thanks for info!